flow/cmd/handler.go

```diff
@@ -193,10 +196,6 @@ func (h *FlowRequestHandler) CreateQRepFlow(
 		workflowFn = peerflow.QRepFlowWorkflow
 	}
 
-	if req.QrepConfig.SyncedAtColName == "" {
-		cfg.SyncedAtColName = "_PEERDB_SYNCED_AT"
```

Comment:
So we don't want this default anymore?

Reply:
We don't, as this prevents having no synced at column via api. Instead ui will be explicit

no impact on clickpipes, as it's only cdc